tests/tck/features/fetch/FetchEdges.strVid.feature

```diff
@@ -249,3 +256,64 @@ Feature: Fetch String Vid Edges
       FETCH PROP ON serve 'Boris Diaw'->'Hawks' YIELD serve.not_exist_prop
       """
     Then a SemanticError should be raised at runtime:
+
+  Scenario: yield edge
+    When executing query:
+      """
+      FETCH PROP ON serve 'Boris Diaw' -> 'Hawks' YIELD serve.start_year, serve.end_year, edge as relationship
+      """
+    Then the result should be, in any order:
+      | serve._src   | serve._dst | serve._rank | serve.start_year | serve.end_year | relationship                                                         |
+      | "Boris Diaw" | "Hawks"    | 0           | 2003             | 2005           | [:serve "Boris Diaw"->"Hawks" @0 {end_year: 2005, start_year: 2003}] |
+    When executing query:
+      """
+      FETCH PROP ON serve "Boris Diaw"->"Spurs" YIELD edge as relationship, src(edge) as src_edge, dst(edge) as dst_edge, type(edge) as type, rank(edge) as rank
+      """
+    Then the result should be, in any order:
+      | serve._src   | serve._dst | serve._rank | relationship                                                         | src_edge     | dst_edge | type    | rank |
+      | "Boris Diaw" | "Spurs"    | 0           | [:serve "Boris Diaw"->"Spurs" @0 {end_year: 2016, start_year: 2012}] | "Boris Diaw" | "Spurs"  | "serve" | 0    |
+    When executing query:
+      """
+      FETCH PROP ON serve "Boris Diaw"->"Not Exist" YIELD src(edge) as a
+      """
+    Then the result should be, in any order:
+      | serve._src | serve._dst | serve._rank | a |
+    When executing query:
+      """
+      FETCH PROP ON like "Tony Parker"->"Tim Duncan", "Grant Hill" -> "Tracy McGrady" YIELD edge as relationship |
```

Comment:
Could we yield `properties(edge) as props` in fetch clause?

Reply:
ok